src/coreclr/jit/jitconfigvalues.h

```diff
@@ -463,6 +460,7 @@ CONFIG_STRING(JitNoInlineRange, W("JitNoInlineRange"))
 CONFIG_STRING(JitInlineReplayFile, W("JitInlineReplayFile"))
 #endif // defined(DEBUG) || defined(INLINE_DATA)
 
+CONFIG_INTEGER(JitExtendedDefaultPolicyModel, W("JitExtendedDefaultPolicyModel"), 1)
```

Comment:
```suggestion
CONFIG_INTEGER(JitExtendedDefaultPolicy, W("JitExtendedDefaultPolicy"), 1)
```
I was using `Model` as an indicator that this policy was based on machine learning models.